src/main/java/org/mifos/processor/bulk/camel/routes/SplittingRoute.java

```diff
@@ -69,18 +72,39 @@ public void configure() throws Exception {
             List<String> subBatchFile = new ArrayList<>();
             Set<String> distinctPayeeIds = transactionList.stream().map(Transaction::getPayeeDfspId).collect(Collectors.toSet());
             logger.info("Payee id {}", distinctPayeeIds);
-            if (partyLookupEnabled && !distinctPayeeIds.isEmpty()) {
+            logger.info("Number of payeeId {}", distinctPayeeIds.size());
+            Boolean batchAccountLookup = (Boolean) exchange.getProperty("batchAccountLookup");
```

Comment:
use variable naming convention to differentiate a boolean variable

Reply:
Updated